### PR TITLE
Backport dashboard-proxy command to 1.18

### PR DESCRIPTION
**Pull request by joedborg**
In order to allow people on `1.18/stable` to access the `dashboard-proxy` command.

### CODE REVIEW

**Comment by ktsakalozos** [comment]
@joedborg this PR does not match what we put into master with https://github.com/ubuntu/microk8s/pull/1198/files

File: scripts/dashboard-proxy.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python3
```
Comment (ktsakalozos) [inline scripts/dashboard-proxy.py:1]: This file is moved under `scripts/wrappers/`

**Comment by joedborg** [comment]
Should all be there now, but had to bring in all the python `black` commit too.

**Comment by ktsakalozos** [comment]
You are right @joedborg, this gets too complicated. I am closing ths in favor of https://github.com/ubuntu/microk8s/pull/1355

**Pull request closed without merging**